.gitignore

```diff
@@ -46,3 +46,4 @@ reporting/cookies.json
 
 #IDEs
 .idea/
+workspace.code-workspace
```

Comment:
Looking at https://code.visualstudio.com/docs/editor/workspaces it seems like the general patterns for VS Code workspaces are `.code-workspace` or `foo.code-workspace`, so we should be able to generalise this `.gitignore` line to `*.code-workspace` I think, to cover all the bases.

Reply:
That would be great — I'll update my filename to `.code-workspace` accordingly